tests/test-parser.js

```diff
@@ -44,23 +44,23 @@ describe('test-parser', function () {
             function() {
                 ks_parser.parse('{{ f({ x: 1 }) }}');
             },
-            /^SyntaxError: .+$/,
+            /^Expected .+ but .+ found\.$/,
```

Comment:
To correctly test for the error type, we’d need to update to [PEG.js&nbsp;v0.8.0](https://github.com/pegjs/pegjs/releases/tag/v0.8.0), which includes https://github.com/pegjs/pegjs/commit/277fb23411477936adcb369d11564d9ef566d74d.